xpk.py

```diff
@@ -2629,14 +2752,24 @@ def directory_path_type(value):
     required=True,
 )
 cluster_create_optional_arguments.add_argument(
-  '--cluster-cpu-machine-type',
+  '--default-pool-cpu-machine-type',
     type=str,
     default='e2-standard-16',
     help=(
-      'Set the machine tpu within the default cpu node pool. For'
+      'Set the machine type within the default cpu node pool. For'
       ' regional clusters, all zones must support the machine type.'
     )
 )
+cluster_create_optional_arguments.add_argument(
+  '--cluster-cpu-machine-type',
```

Comment:
Do we really want to deprecate this field?

Reply:
Yes, I hope to minimize confusion by having better args names, especially since I am introducing CPUs. This field is deprecated, to be replaced by "--default-pool-cpu-machine-type"

Reply:
I agree it can be confusing for the --device-type=CPU_TYPE user. 

Since we added deprecated messages, the impact would be that any current user would have to change their argument. How about we set this to a 'warning' level for now so the old argument still works. We want to warn users still with a print statement during xpk execution, and also in the help argument. In the future we can delete it completely after users are more aware that it is being deprecated.

Reply:
Sure, sounds good, will update the print message to have a warning about deprecation.